docs/ensnode.io/src/content/docs/docs/contributing/releases.mdx

```diff
@@ -7,41 +7,127 @@ sidebar:
+
+To test or install a package before merging it into `main`, a preview release can be used. To manually trigger a preview release for a branch, follow these steps:
+
+1. Navigate to [Actions > Release Preview](https://github.com/namehash/ensnode/actions/workflows/release_preview.yml)
+2. Click "Run workflow" and select from the following options:
+ - The branch on which to run the preview release workflow. Any branch other than `main` can be selected. Using a branch with an active PR allows the workflow to add a GitHub comment containing the installation information. If multiple preview releases are triggered for the same brach/PR, the GitHub comment will update with the latest release.
+ - Choose which artifacts to build and publish (NPM packages, Docker images, Lambda zips, or any combination).
+ - (Optional) Provide a custom suffix for the preview release tag. For example, if you had a branch called `feat/add-api-route` and left this custom suffix field blank, the preview release would be `@ensnode/[package-name]@preview-feat-add-api-route`. If you fill in the custom suffix field with `users-route` then the resulting tag name would be `@ensnode/[package-name]@preview-users-route`.
+3. Install preview packages with: `npm install @ensnode/[package-name]@preview-branch-name`. 
+
+**Important notes:**
+- Preview releases are generally built and published from branches with active PRs. They are not guaranteed to be stable as they are still under active development.
+- Preview releases can only be triggered manually.
+- Preview releases will include the `@preview-*` tag for published artifacts, followed by either the name of the branch or the custom suffix chosen during the action trigger.
+- Published artifacts will advance to the version that was included in the changeset of the selected branch.
+- Preview releases can only be triggered by authorized NameHash team members.
+- No GitHub releases or tags are created for Preview Releases.
+
+# Selecting a Release for Deployment or Installation
+
+When using ENSNode artifacts, you have several release types to choose from.
+
+## Where to Find Releases
+
+- **NPM Packages**: Available on the [npm registry](https://www.npmjs.com/search?q=%40ensnode) under the `@ensnode` organization
+- **Docker Images**: Available on [GitHub Container Registry](https://github.com/orgs/namehash/packages?repo_name=ensnode)
+- **GitHub Releases**: Full releases are documented with release notes on the [ENSNode GitHub releases page](https://github.com/namehash/ensnode/releases)
+- **Lambda Zip Artifacts**: Available in the Artifact section of a successful workflow run
+
+## Choosing the Right Release Type
+
+### Pinned Full Release Versions
+:::note
+The example version below of `1.0.0` is a placeholder. Visit the [Releases Page](https://github.com/namehash/ensnode/releases) to pick a pinned full release.
+:::
+```bash
+npm install @ensnode/[package-name]@1.0.0
```

Comment:
Switched to `[package-name]` and `ensnode/[app-name]` per suggestion from @shrugs